src/api/analytics.ts

```diff
@@ -54,7 +57,11 @@ export type AnalyticsResponse = {
         hidden: boolean;
         meta: boolean;
     }>;
-
+    metaData: {
+        dimensions: Record<string, string[]>;
+        items: Record<string, { name: string; uid?: Id; code?: string; options: any[] }>;
+        pager: Pager;
+    };
```

Comment:
when `skipMeta=true`, this object is empty, so : {} | {...}. Also, I didn't get a pager on test request I did, so let's put it optional.

It's hard to get the type exact, as the options influence the shape of the response, but for now this should be enough.